exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>This may be a naive comment because my background is with so much longer wavelengths than UV ;-) But could the title be more descriptive? I will read on to see if I have a suggestion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>